pipeline/processes/shaping/shaping.C

```diff
@@ -0,0 +1,74 @@
+Int_t shaping() {
+    TRestRawSignalEvent* ev = new TRestRawSignalEvent();
+
+    TRestRawSignal* sgnl = new TRestRawSignal();
+    for (int n = 0; n < 512; n++) sgnl->AddPoint(0);
```

Comment:
I think TRestRawSignal is initialized to zero in the constructor, so I think you can remove this line.

Reply:
ok